daemon/api.go

```diff
@@ -1168,3 +1177,76 @@ func assertsFindMany(c *Command, r *http.Request) Response {
 func getEvents(c *Command, r *http.Request) Response {
 	return EventResponse(c.d.hub)
 }
+
+type changeInfo struct {
+	Kind    string      `json:"kind"`
+	Summary string      `json:"summary"`
+	Status  string      `json:"status"`
+	Tasks   []*taskInfo `json:"tasks,omitempty"`
+}
+
+type taskInfo struct {
+	Kind    string   `json:"kind"`
+	Summary string   `json:"summary"`
+	Status  string   `json:"status"`
+	Log     []string `json:"log,omitempty"`
+}
+
+func getChanges(c *Command, r *http.Request) Response {
+	query := r.URL.Query()
+	logTail := 1
+	qLogTail := query.Get("log-tail")
```

Comment:
Sorry, I meant the opposite: I suggest returning everything we have in memory, instead of offering control. The issue of not overflowing the client and not overflowing the memory is similar, so we don't need a flag. If we ever change the backend to keep all logs, we can move the decision of how much to keep here, so it's sane by default.